guides/common/modules/proc_configuring-the-templatesync-plugin.adoc

```diff
@@ -5,7 +5,7 @@ In the {ProjectWebUI}, navigate to *Administer* > *Settings* > *TemplateSync* to
 The following table explains the attributes behavior.
 Note that some attributes are used only for importing or exporting tasks.
 
-.Synchronizing Templates Plug-in configuration
+.Synchronizing templates plug-in configuration
```

Comment:
I think we shouldn't decapitalize plug-in/feature names, such as Templates, Registration, etc. Perhaps even add this to the capitalization list?

Reply:
![image](https://github.com/theforeman/foreman-documentation/assets/20555018/7e9e3d28-8df9-4294-883c-e296a76e053a)

Reply:
This PR only covers general text, not UI elements. 

The UI must be documented accurately but we cannot use the UI as a guideline for capitalization. For example, in this list "Openscap" should probably be "OpenSCAP".

Reply:
I am going to fix a couple other capitalization issues. Also, [RHSSG](https://redhat-documentation.github.io/supplementary-style-guide/#_o) says not to use "OS". It is supposed to be written out in full as "operating system".

Reply:
Oh, I see. Then perhaps the wording isn't ideal. Could we change this heading to "Template synchronization plug-in configuration" ?

Reply:
@Lennonka Can you confirm the plug-in name?

Reply:
Going by the title of this module, it should be the TemplateSync plug-in.

Reply:
I agree, especially since the UI seems to use the term. I will update this module.